start-from-zero/index.css

```diff
@@ -44,27 +94,50 @@ body, html {
 form {
     display: flex;
     flex-direction: column;
-    
     background: white;
     padding-top: 30px;
     padding-bottom: 30px
 }
 
 .blue-line {
     text-align: center;
-    height: 50px;
-    line-height: 50px;
     background: #03A9F4;
+    height: 50px;
+    line-height: 48px;
     color: white;
+    font-size: 16px;
+    -webkit-box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    box-shadow: 0px 3px 15px -2px rgba(0,0,0,0.79);
+    position: relative;
+    z-index: 10000;
+}
+
+.form-column{
+    -webkit-box-shadow: 0px 0px 6px 0px rgba(0,0,0,0.75);
+-moz-box-shadow: 0px 0px 6px 0px rgba(0,0,0,0.75);
```

Comment:
Feel free to setup your environment however you're most comfortable. If it increases your productivity, I see absolutely no reason why you shouldn't make use of such tooling (and maybe even share with your colleagues and even us, trainers - believe it or not, we don't know _absolutely everything_ either :stuck_out_tongue:), just as long as you're able to fully grasp the base concepts that such tooling eases.